### ISSUE TITLE
An error occurred while loading the routes definition of redmine_git_hosting plugin

### ISSUE THREAD

**Issue by ymolinet** [jbox-web/redmine_git_hosting#326]
I'm just update my redmine by using git repository.
I'm update the plugin as mentionned in the how to.
I'm install ruby 2 on my debian.

1. bundle install give an error because plugin and redmine have a common dependency (redcarpet).
Redmine require 3.1.2 and the plugin require 2.3.0. 
Just update the Gemfile ?

2. when i run 
 RAILS_ENV=production NAME=redmine_git_hosting rake redmine:plugins:migrate
I receive 
```
/var/www/clients/client1/web6/web/plugins/redmine_git_hosting/lib/tasks/selinux.rake:52: warning: class variable access from toplevel
/var/www/clients/client1/web6/web/plugins/redmine_git_hosting/lib/tasks/selinux.rake:283: warning: class variable access from toplevel
/var/www/clients/client1/web6/web/plugins/redmine_git_hosting/lib/tasks/selinux.rake:284: warning: class variable access from toplevel
/var/www/clients/client1/web6/web/plugins/redmine_git_hosting/lib/tasks/selinux.rake:285: warning: class variable access from toplevel
An error occurred while loading the routes definition of redmine_git_hosting plugin (/var/www/clients/client1/web6/web/plugins/redmine_git_hosting/config/routes.rb): You should not use the `match` method in your router without specifying an HTTP method.
If you want to expose your action to both GET and POST, add `via: [:get, :post]` option.
If you want to expose your action to GET, use `get` in the router:
  Instead of: match "controller#action"
  Do: get "controller#action".
```

**Comment by n-rodriguez** [comment]
> Redmine require 3.1.2 and the plugin require 2.3.0.

mmm... Which version of Redmine are you running? Because as far as I know Redmine 2.6 still uses Redcarpet 2.3 (http://www.redmine.org/projects/redmine/repository/entry/branches/2.6-stable/Gemfile#L37)

About the route definition it will be fixed for the next versions (v0.7.10 and v1.0)

**Comment by n-rodriguez** [comment]
The routes definition loading is fixed in commit https://github.com/jbox-web/redmine_git_hosting/commit/e8d1d17ccc6ee01e3a584b1ce839d71587e2dfc5

**Comment by ymolinet** [comment]
In https://github.com/redmine/redmine/blob/master/Gemfile
Redcarpet Version if 3.1.2

**Comment by ymolinet** [comment]
An error occurred while loading the routes definition of redmine_graphs plugin (/var/www/clients/client1/web6/web/plugins/redmine_graphs/config/routes.rb): You should not use the `match` method in your router without specifying an HTTP method.
same error:

If you want to expose your action to both GET and POST, add `via: [:get, :post]` option.
If you want to expose your action to GET, use `get` in the router:
  Instead of: match "controller#action"
  Do: get "controller#action".

**Comment by vanch** [comment]
Migrated from redmine 2.4.3 to 3.3.4 with 0.7 git_hosting plugin version and this commit helped with the same issue, thx.

> The routes definition loading is fixed in commit e8d1d17
